src/App.js

```diff
@@ -1,28 +1,20 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
 
 class App extends Component {
+  state = { blocks: ['hello world', 'another string'] };
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
-          <a
-            className="App-link"
-            href="https://reactjs.org"
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            Learn React
-          </a>
-        </header>
+      <div>
+        {this.state.blocks.map(block => {
+          return <p key={block}>{block}</p>;
+        })}
       </div>
     );
   }
 }
 
 export default App;
+
+// figure out how to render a blocks object using either a p tag or an h1 tag.
+// hint: switch statement that returns a different component based on value inside block object
```

Comment:
```js
// another hint: each block should probably look something like this:
{
  type: "heading",
  text: "hello world", // or whatever
}, {
  type: "text",
  text: "another string",
}
```

That should be enough to get started … we can figure out how to extend this when we meet on Monday (assuming that still works). We could do `type: "unordered-list"` or `type: "image"` or maybe adding attributes to blocks maybe? I think doing inline-block stuff (e.g. bold, italic, link, etc) should wait. That's going to be a little more complicated.

After we get a few block types in here we can refactor to split each out into it's own component and maybe take a look at UI for adding new blocks and moving blocks around. Also, testing would be good.